emmet-core/emmet/core/qchem/calculation.py

```diff
@@ -0,0 +1,669 @@
+    )
+    calc_type: CalcType = Field(
+        None,
+        description="Combination dict of LOT + TaskType: B97-D/6-31g*/VACUUM Geometry Optimization",
+    )
+
+    @classmethod
+    def from_qchem_files(
+        cls,
+        dir_name: Union[Path, str],
+        task_name: str,
+        qcinput_file: Union[Path, str],
+        qcoutput_file: Union[Path, str],
+        store_energy_trajectory: bool = False,
+        qcinput_kwargs: Optional[Dict] = None,
+        qcoutput_kwargs: Optional[Dict] = None,
+    ) -> "Calculation":
+        """
+        Create a QChem calculation document from a directory and file paths.
+
+        Parameters
+        ----------
+        dir_name
+            The directory containing the QChem calculation outputs.
+        task_name
+            The task name.
+        qcinput_file
+            Path to the .in/qin file, relative to dir_name.
+        qcoutput_file
+            Path to the .out/.qout file, relative to dir_name.
+        store_energy_trajectory
+            Whether to store the energy trajectory during a QChem calculation #TODO: Revisit this- False for now.
+        qcinput_kwargs
+            Additional keyword arguments that will be passed to the qcinput file
+        qcoutput_kwargs
+            Additional keyword arguments that will be passed to the qcoutput file
+
+        Returns
+        -------
+        Calculation
+            A QChem calculation document.
+        """
+
+        dir_name = Path(dir_name)
+        qcinput_file = dir_name / qcinput_file
+        qcoutput_file = dir_name / qcoutput_file
+
+        output_file_paths = _find_qchem_files(dir_name)
+
+        qcinput_kwargs = qcinput_kwargs if qcinput_kwargs else {}
+        qcinput = QCInput.from_file(qcinput_file, **qcinput_kwargs)
+        # print(qcinput)
+
+        qcoutput_kwargs = qcoutput_kwargs if qcoutput_kwargs else {}
+        qcoutput = QCOutput(qcoutput_file, **qcoutput_kwargs)
+
+        completed_at = str(datetime.fromtimestamp(qcoutput_file.stat().st_mtime))
+
+        input_doc = CalculationInput.from_qcinput(qcinput)
+        output_doc = CalculationOutput.from_qcoutput(qcoutput)
+
+        has_qchem_completed = (
+            QChemStatus.SUCCESS
+            if qcoutput.data.get("completion", [])
+            else QChemStatus.FAILED
+        )
+
+        if store_energy_trajectory:
+            print("Still have to figure the energy trajectory")
+
+        return cls(
+            dir_name=str(dir_name),
+            task_name=task_name,
+            qchem_version=qcoutput.data["version"],
+            has_qchem_completed=has_qchem_completed,
+            completed_at=completed_at,
+            input=input_doc,
+            output=output_doc,
+            output_file_paths={
+                k.lower(): Path(v)
+                if isinstance(v, str)
+                else {k2: Path(v2) for k2, v2 in v.items()}
+                for k, v in output_file_paths.items()
+            },
```

Comment:
Since each `Calculation` document describes one specific Q-Chem calc, with a particular input and output file, it seems odd to me to store the complete list of output file paths here. Doesn't that belong at the Task Document level?